packages/react-scripts/config/jest/babelTransform.js

```diff
@@ -6,10 +6,23 @@
  * LICENSE file in the root directory of this source tree.
  */
 'use strict';
-
 const babelJest = require('babel-jest');
+const paths = require('../paths');
 
-module.exports = babelJest.createTransformer({
+const babelTransformer = babelJest.createTransformer({
   presets: [require.resolve('babel-preset-react-app')],
   babelrc: false,
 });
+
+// choose files to transpile
+// jest < 22.0.x transform patterns don't resolve symlinks, ie. don't match against realpaths
+// jest >= 22.0.04 matches against realpaths, but this method might be preferable anyway
+const babelProcess = babelTransformer.process;
+babelTransformer.process = (src, filename, config, transformOptions) => {
```

Comment:
I'd prefer to remove this (and rely on Jest 22 behavior). After ejecting, the project is already complex enough. We should try to reduce the area for complexity that the user will have to maintain directly.